gxtest/cgx.h

```diff
@@ -66,7 +66,7 @@ void CGX_LoadProjectionMatrixOrthographic(float mtx[4][4]);
 
 void CGX_DoEfbCopyTex(u16 left, u16 top, u16 width, u16 height, u8 dest_format,
                       bool copy_to_intensity, void* dest, bool scale_down = false,
```

Comment:
This should probably be refactored to take the optional parameters as a parameter struct of `std::optional`s instead. Otherwise, writing testing code with a particular behavior in mind becomes too error-prone, and reading testing code and figuring out what actual state is being set up becomes too confusing.

Reply:
Probably a struct that contains default values in it and is set up using brace initialization would work better than `std::optional`s; that way the struct itself has the default values instead of `CGX_DoEfbCopyTex`.

Reply:
True, sounds reasonable :)